CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Cromwell Change Log
 
+## 40 Release Notes
+
+### Bug fixes
+
+#### Allow a WDL call with no inputs to have an empty `{ }` block
+
+In Cromwell 39 and below, the following syntax was erroneously rejected by Cromwell's WDL 1.0 implementation:
```

Comment:
Can we compress these two sentences down to just:
> Cromwell now correctly allows empty call blocks (eg `call task_with_no_inputs {}`) in WDL 1.0+, as addition to existing support in draft-2 

?

Reply:
This came out rather strange sounding the first time around, have pushed a nicer version